packages/shorebird_cli/lib/src/commands/patch/android_patcher.dart

```diff
@@ -149,6 +150,7 @@ Looked in:
           releaseArtifactPath: releaseArtifactPath.value,
           patchArtifactPath: patchArtifactPath,
         );
+
```

Comment:
Is this from your autoformatter?

Reply:
Not really, I guess I accidentally added it.

Removing it

```suggestion
```